source/js/payments-card.js

```diff
@@ -51,6 +63,50 @@ function setupBraintree() {
     container.classList.remove(braintreeErrorClass);
   }
 
+  function showDetailFieldError(field) {
+    // If field error is already showing, do not render again
+    if (!field.parentNode.classList.contains("form-item--errors")) {
+      field.parentNode.classList.add("form-item--errors");
+      document
+        .getElementById(`error-message__` + `${field.name}`)
+        .classList.remove("hidden");
+    }
+  }
+
+  function checkYourDetailsFields() {
```

Comment:
Function that loops through all required fields and checks them. If any are not filled out, it will then display the error state.